OPERATOR.rst

```diff
@@ -231,14 +231,18 @@ To do a promotion:
 Backporting from ``prod`` to ``develop``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-#. Open a PR from the GitHub UI titled::
+#. Make a branch from ``prod`` at the last commit being backported. Title the
+   branch following this pattern::
+
+       issues/<your name>/backport-<hash of hotfix>-<another hotfix>
```

Comment:
s/<your name>/<your GH handle> maybe?

Reply:
I use my name, so I don't think that's a hard and fast rule. I'll leave it as is.